samples/azure-functions/service-bus-kafka/sample.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Using NServiceBus and Kafka in Azure Functions (isolated)
+summary: Using NServiceBus with Kafka triggers and Azure Functions isolated worker hosting model.
+component: ASBFunctionsWorker
+related:
+ - nservicebus/hosting/azure-functions-service-bus
+redirects:
+ - samples/previews/azure-functions/kafka
+reviewed: 2023-09-13
+---
+
+This sample shows how to process Kafka events using an Azure Functions trigger and how to follow up on those events using an NServiceBus SendOnly endpoint in an Azure Function.
+
+### Kafka and NServiceBus
+
+[Kafka](https://kafka.apache.org/) is an event streaming broker, similar to [Azure Event Hubs](https://azure.microsoft.com/en-us/products/event-hubs). Event streaming brokers are used to store massive amounts of events that are coming in, for example from IoT devices. NServiceBus on the other hand works on top of messaging brokers like Azure Service Bus, RabbitMQ and Amazon SQS/SNS. They can complement each other as shown in this sample, which has two projects.
+
+- A ConsoleEndpoint is the starting point of the sample, which products numerous events to Kafka.
+- An Azure Function using a Kafka trigger to consume the events and at a certain point sends a message using an NServiceBus SendOnly endpoint back to the ConsoleEndpoint.
+
+For more information about Kafka and NServiceBus read the blogpost [Let's talk about Kafka](https://particular.net/blog/lets-talk-about-kafka).
+
+downloadbutton
+
+## Prerequisites
+
+### Configure Connection string
+
+To use the sample a valid Service Bus connection string must be provided
+
+- in the `local.settings.json` file in the `AzureFunctions.KafkaTrigger.FunctionsHostBuilder` project.
+
+- as an environment variable named `AzureServiceBus_ConnectionString`.
+
+### Kafka broker
+
+This sample requires Kafka available to store the events being produced.
+
+#### Set up Kafka Docker container
```

Comment:
I wonder if we could call docker-compose up in the start up of the console? To improve the F5 experience? With a comment stating don't do this in production :) Could be a separate app in the sln that would do that. I may try to see if I can make this work.